python/dgl/dataloading/dataloader.py

```diff
@@ -678,3 +680,50 @@ def collate(self, items):
             return self._collate(items)
         else:
             return self._collate_with_negative_sampling(items)
+
+class GraphCollator(object):
+    """DGL collator to combine graphs and their computation dependencies within a minibatch for
+    training graph classification or regression.
```

Comment:
There's no concept of "computation dependency" for graph classification/regression.
```
DGL collator to combine graphs as well as their graph-level data into a batched graph
for training graph classification or regression.
```
If you do implement the more general collate function:
```
Given a set of graphs as well as their graph-level data, the collate function will batch the
graphs into a batched graph, and stack the tensors into a single bigger tensor.  If the
example is a container (such as sequences or mapping), the collate function preserves
the structure and collates each of the elements recursively.
```
(See also https://pytorch.org/docs/stable/data.html#working-with-collate-fn)